converter/stitching/stitcher.go

```diff
@@ -110,7 +112,8 @@ func (s *stitcher) stitchFlow(flow input.Flow) error {
 	//AggregateQuery in the matcher, and the
 	//sessions qualify for merging/ stitching
 	matchFound := false
-
+	matchAgg := false
```

Comment:
matchAgg should be a session.Aggregate

It will hold the result of our search for the aggregate with the lowest matching cost.

Reply:
where will this matchAgg be used in code? I added it but it gives me error that its defined but not used. I committed the code with it